tutorials/javaFxPart1.md

```diff
@@ -109,6 +109,30 @@ public class Duke extends Application {
 }
 ```
 
+However, if you do already have an existing constructor which takes arguments, you can proceed by creating an overloaded constructor with no arguments, passing the arguments to the existing constructor e.g.,
```

Comment:
Based on the current flow, we are creating a no-argument constructor before this. 
It may be clearer to frame this as creating a new constructor that accepts arguments, in addition to the no-argument constructor. 
Then, tell the reader about refactoring the no-argument constructor to initialize the object with default values and then allowing the argument-based constructor to handle custom initialization.

or

change the above pre-existing explanation and code to create an argument-based constructor first to keep the flow consistent

Reply:
Thanks for the review, @NeoHW, I've made changes to frame the case where there are arguments in the constructor as a tip, so that it will be more apparent to users that this only applies in the case that they have existing main classes which have constructors with arguments.